features/step_definitions/snippet_steps.rb

```diff
@@ -15,10 +16,68 @@
   click_button "submit_snipet"
 end
 
-Then(/^I should see a success message$/) do
-  expect(page).to have_content("Snipet was successfully created.")
+Then(/^I should see a "(.*?)" message$/) do |message_type|
+  if message_type.equal? "success"
+    expect(page).to have_content("Snipet was successfully created.")
+  elsif message_type.equal? "update success"
+    expect(page).to have_content("Snipet was successfully updated.")
+  elsif message_type.equal? "update delete"
+    expect(page).to have_content("Snipet was successfully destroyed.")
+  end
 end
 
 Then(/^I should see my snippets page$/) do
   expect(current_path).to match(/users\/\d+\/snipets/)
 end
+
+When(/^I go to my snippets page$/) do
+  visit new_snipet_path
+  fill_in "snipet_title", :with => "my first snippet"
+  fill_in "snipet_content", :with => "ls"
+  click_button "submit_snipet"
+
+  #FIXME: How can I get current_user id, in place of writing id ?
+  visit my_snipets_path(3)
```

Comment:
you can save the user in the "Given(/^I am authenticated$/)" step in an instance variable

``` ruby
Given(/^I am authenticated$/) do
  @user = ...
end
```

and then use it in this step with:

``` ruby
visit my_snipets_path(@user.id)
```

The trick here is that all the steps share a common object when your test executes